src/components/Navbar.js

```diff
@@ -20,7 +20,7 @@ const Navbar = () => {
           {links.map((link) => (
             <React.Fragment key={link.text}>
               <li>
-                <NavLink to={link.path}>{link.text}</NavLink>
+                <NavLink to={link.path} className={styles.link}>{link.text}</NavLink>
               </li>
```

Comment:
### Impressive job @AyoMoses1, I just have a couple of suggestions to make, 
- [ ] As you can see from Zeplin design there is a user Icon, which is missing from your project. Please add this icon with the appropriate styling. This is to make sure you have the exact copy of the design. 
- [ ] Also there is a vertical line between the `comment, Remove, edit` texts, which are under the book title, Besides that, Notice only some of the text is blurred than the others and only the book title is bold. Please take a look and make the improvements. 

### Zeplin design
![Screen Shot 2023-03-10 at 5 08 16 AM](https://user-images.githubusercontent.com/69077061/224324108-be786c25-d72c-405a-9bf7-acc1beb36761.png)
### Your design 
![Screen Shot 2023-03-10 at 5 10 27 AM](https://user-images.githubusercontent.com/69077061/224324436-f0b78d6b-80c8-4437-a2a5-539dde65b151.png)

Reply:
Thanks so much @TadesseDev . It was an oversight from my part. I really appreciate

Reply:
Your doing great @AyoMoses1 ✅